CondFormats/HcalObjects/interface/alpaka/HcalRecoParamWithPulseShapeDevice.h

```diff
@@ -0,0 +1,29 @@
+#ifndef CondFormats_HcalObjects_interface_alpaka_HcalRecoParamWithPulseShapeDevice_h
+#define CondFormats_HcalObjects_interface_alpaka_HcalRecoParamWithPulseShapeDevice_h
+
+#include "CondFormats/HcalObjects/interface/HcalRecoParamWithPulseShapeHost.h"
+#include "CondFormats/HcalObjects/interface/HcalRecoParamWithPulseShapeSoA.h"
+#include "DataFormats/Portable/interface/alpaka/PortableCollection.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/memory.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+  using HcalRecoParamWithPulseShapeDevice = HcalRecoParamWithPulseShapeT<Device>;
+}
+
+namespace cms::alpakatools {
+  template <>
+  struct CopyToDevice<HcalRecoParamWithPulseShapeHost> {
```

Comment:
It would be better to define this specialization of `CopyToDevice` in `CondFormats/HcalObjects/interface/HcalRecoParamWithPulseShapeHost.h`.

Reply:
Moved to the host file in d82fbab